### ISSUE TITLE
Change of GET parameters are not being pushed to history

### ISSUE THREAD

**Issue by daverdalas** [router5/router5#133]
Hey, I've been using router5 lately. Everything works as excepted I only have one problem. When I replace GET parameters they are not added to the browser history. I do not know how to make this work. When i hit previous button in browser I want go previous parameters not to previous page. When I change a page using router everything works ok but not when changing parameters.

My router instance:
```js
const router = Router5([
                    {
                        name: 'existingCity', path: '/drwale-*citySlug'
                    },
                    {
                        name: 'notExistingCity', path: '/drwale'
                    }
                ], {
                    strictQueryParams: false,
                }).usePlugin(browserPlugin({}));
router.start();
```
Parameters change:
```js
vm.router.navigate('existingCity', parameters, {reload: true});
```
Url changes properly, only thing that is missing is new history entry.

**Comment by motet-a** [comment]
You can put the parameters in the URL without splat parameters, even if it doesn't really solve your problem. In the routes:

```js
[
    {
        name: 'city', path: '/cities/:cityId'
    },
]
```

And then:

```js
// Navigates to /cities/paris
router.navigate('city', { cityId: 'paris' })

// Navigates to /cities/marseille
router.navigate('city', { cityId: 'marseille' })

// If you press the 'previous' button in your browser here, you 
// go back to Paris
```

**Comment by daverdalas** [comment]
@motet-a I have no problem with splat parameter. This part works ok. The problem is with GET parameters, which change does not add a history entry.

**Comment by troch** [comment]
@daverdalas I think I have fixed it in version 5.1.2
